src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__description {
+    margin: 0 0 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-review {
```

Comment:
`rating-review` is an element of `card` block. All elements must wrote inside a parent block element. Inside elements, we write pseudo-classes or pseudo-elements.

Reply:
Removed span as a redundant tag. Moved typography styles to card__rating ✅